### PR TITLE
Remove outdated docs on ledger supporting extrinsic for setting identity

### PR DESCRIPTION
**Pull request by platonicsocrates**
We now have support for setting identity using ledger!

See https://github.com/ZondaX/ledger-polkadot#identity

### CODE REVIEW

**Review by DrW3RK — COMMENTED** [review]
Thank you! I'm wondering if the note should still be up for Ledger Nano S users.

File: docs/learn/learn-identity.md [hunk 1]
```diff
@@ -39,13 +39,6 @@ The procedure to set and clear identities is explained in detail in this support
 
 :::
 
-:::note
-
-The Ledger app doesn't support the extrinsic for setting identity yet. As a workaround, create a
-primary identity with an on-chain account and then using that primary identity, assign a
-[sub-identity](#sub-accounts) to the Ledger stash.
-
-:::
```
Comment (profradha) [inline docs/learn/learn-identity.md]: The Ledger app on **Nano S** doesn't support the extrinsic for setting identity. As a workaround, create a
primary identity with an on-chain account and then using that primary identity, assign a
[sub-identity](#sub-accounts) to the Ledger stash.

File: docs/maintain/maintain-guides-validator-community.md [hunk 2]
```diff
@@ -56,12 +56,7 @@ means of reaching out. Nominators may wish to know more about the Validator, the
 future staking plans, tooling used, or several additional topics. Having a dedicated website
 additionally to provide this sort of information is ideal.
 
-:::note Ledger app doesn't support the extrinsic for setting identity yet
```
Comment (profradha) [inline docs/maintain/maintain-guides-validator-community.md:59]: Ledger app on Nano S doesn't support the extrinsic for setting identity

**Comment by platonicsocrates** [comment]
> Thank you! I'm wondering if the note should still be up for Ledger Nano S users.

great point! I'll update that

File: docs/learn/learn-identity.md (hunk 1 above)
Comment (platonicsocrates) [reply docs/learn/learn-identity.md]: updated!

File: docs/maintain/maintain-guides-validator-community.md (hunk 2 above)
Comment (platonicsocrates) [reply docs/maintain/maintain-guides-validator-community.md:59]: updated!

**Review by DrW3RK — APPROVED** [review]
Thank you very much!

**Pull request merged**